index.html

```diff
@@ -205,48 +205,62 @@ <h2>Introduction</h2>
         <h3>Design Goals and Rationale</h3>
 
         <p>
-The Data Integrity specification achieves the following design goals:
+This specification optimizes for following design goals:
         </p>
 
         <dl>
-          <dt>Simple for Developers</dt>
+          <dt>Simplicity</dt>
           <dd>
-The proof format is designed to be easy to use for developers that don't
-have significant cryptography training. For example, <a>cryptographic suite</a>
-identifiers are used instead of specific cryptographic parameters to ensure
-that it is difficult to accidentally produce a weak digital proof.
+The technology is designed to be easy to use for application developers, who
+often don't have significant training in cryptography. It optimizes for the
+following priority of constituencies: application developers over cryptographic
+suite implementers, over cryptographic suite designers, over cryptographic
+algorithm specification authors.  The solution focuses on sensible defaults and
+preventing the selection of ineffective protection mechanisms. See section
+<a href="#protecting-application-developers"></a> and
+<a href="#versioning-cryptography-suites"></a> for further details.
           </dd>
-          <dt>Layered Architecture</dt>
+          <dt>Composability</dt>
           <dd>
 A number of historical digital signature mechanisms have had monolithic
-designs which limited use cases by combining data normalization, syntax,
+designs which limited use cases by combining data transformation, syntax,
 digital signature, and serialization into a single specification. This
 specification layers each component such that a broader range of use cases,
 such as generalized selective disclosure and serialization-agnostic signatures,
-are enabled.
+are enabled. See section <a href="#transformations"></a>, section
```

Comment:
```suggestion
specification layers each component such that a broader range of use cases are
enabled, including generalized selective disclosure and serialization-agnostic
signatures. See section <a href="#transformations"></a>, section
```

Reply:
@TallTed, I can't apply this change because it's a change to text that was also modified by @dlongley. I got some new and very interesting errors from Github that I've never seen before. Basically, suggesting changes to text that other people have also suggested changes to makes it impossible for Github to reconcile applying both changes (even if you try to stage the changes in a logical patch order that should work). I have applied this change manually:  b0178e865bb6ea3874b1c7822fccc3f79e7bcc59

Reply:
Sorry, @msporny, I thought I had included @dlongley's changes in my own, so you should only have had to accept my suggestion... but maybe that breaks GitHub's contribution records. I'm fine with manual application of changes in such cases, and apologize for the extra effort required.